helix-core/src/main/java/org/apache/helix/task/GenericTaskRebalancer.java

```diff
@@ -40,18 +40,18 @@
 
   @Override
   public Set<Integer> getAllTaskPartitions(JobConfig jobCfg, JobContext jobCtx,
-      WorkflowConfig workflowCfg, WorkflowContext workflowCtx, WorkflowControllerDataProvider cache) {
-    return taskAssignmentCalculator
-        .getAllTaskPartitions(jobCfg, jobCtx, workflowCfg, workflowCtx, cache.getIdealStates());
+      WorkflowConfig workflowCfg, WorkflowContext workflowCtx,
```

Comment:
Same here. Shall we remove the rebalancer related class. Or, at least, deprecate it.

Reply:
Removed.